apps/native/app/src/screens/document-detail/document-detail.tsx

```diff
@@ -260,20 +252,8 @@ export const DocumentDetailScreen: NavigationFunctionComponent<{
     setVisible(true)
   })
 
-  useNavigationComponentDidDisappear(() => {
-    setVisible(false)
-    if (hasPdf) {
-      setLoaded(false)
-    }
-    if (touched) {
-      Navigation.updateProps(ComponentRegistry.InboxScreen, {
```

Comment:
This was causing an unnecessary refetch in the inbox component and more importantly cause FlashList error in Android